src/Libraries/RevitNodes/Elements/InternalUtilities/ElementQueries.cs

```diff
@@ -95,34 +88,32 @@ filtering to get our intended element set.
             if (ClassFilterExceptions.Contains(elementType))
             {
                 var type = GetClassFilterExceptionsValidType(elementType);
-                return new Autodesk.Revit.DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
+                return new DB.FilteredElementCollector(DocumentManager.Instance.CurrentDBDocument)
                     .OfClass(type)
                     .Where(x => x.GetType() == elementType)
-                    .Select(x => ElementSelector.ByElementId(x.Id.Value))
```

Comment:
There is an additional validation done by `ElementSelector.ByElementId` that gets removed by the new code. In theory this could propagate errors further down the Dynamo graph, for example a dangling wrapped element from a previous graph run that since got deleted in Revit or maybe elements from a linked model.

It's hard to say the impact removing this check will have, especially since we don't yet have code coverage on D4R - that would help a lot when we will have it - but at least I think this is worth a bit of consideration and thoughtful testing.

Reply:
True, when getting trace data, that check will be important. Luckily in this particular case we are only wrapping existing Revit elements that we just collected.